conda/recipes/cuspatial/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Comment:
for consistency, shouldn't `cuda{{cuda_major}}` be at the beginning, like cudf has: https://github.com/rapidsai/cudf/blob/branch-23.08/conda/recipes/cudf/meta.yaml#L19

Reply:
No strong opinion

It shouldn't matter as long as the 2 CUDA versions are distinct

Reply:
```suggestion
  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```